pkgs/by-name/ti/tinyfetch/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  stdenv,
+  gcc,
+  fetchFromGitHub,
+  lib,
+}:
+
+stdenv.mkDerivation {
+  pname = "tinyfetch";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "abrik1";
+    repo = "tinyfetch";
+    rev = "c23c22d8af6cf60f15e18c9859623e2faff03dbc";
+    hash = "sha256-nuC7Xtfg7rf6/IDhUKT2JZD9V0Hl/welNAFDJYJKwmA=";
+  };
+
+  buildInputs = [ gcc ];
+
+  buildPhase = ''
+    cd $src/src
+    mkdir -p $out/bin
+    gcc tinyfetch.c -o $out/bin/tinyfetch
```

Comment:
Won't that build without any optimization (which generates awful code)? Shouldn't you add `-O2` here?

Reply:
If upstream considers it as ideal (https://github.com/abrik1/tinyfetch/blob/main/Makefile#L4) I don't think we have to do more about it.